testing/matrix/linter/rules/rules.go

```diff
@@ -177,13 +197,35 @@ func containerImageTagLatest(object storage.StoreObject, containers []v1.Contain
 				"Can't parse an image for container",
 			)
 		}
-		if imageParts[1] == "latest" {
+
+		if imageParts[0] != defaultRegistry {
 			return errors.NewLintRuleError("CONTAINER003",
 				object.Identity()+"; container = "+c.Name,
 				nil,
-				"Image tag \"latest\" used",
+				"Image name should be "+defaultRegistry,
 			)
 		}
+
+		r := regexp.MustCompile("[A-Fa-f0-9]{64}")
+		switch os.Getenv("CI") {
+		case "true":
+			matched := r.MatchString(imageParts[1])
```

Comment:
`MatchString` returns `true` for all strings with length >= 64. It would be best if you wrapped your regexp in `$^` to make it a full match.

Reply:
Yes, good point, thanks !